### ISSUE TITLE
Please expose CURLOPT_PIPEWAIT in the curl tool

### ISSUE THREAD

**Issue by TvdW** [curl/curl#4494]
# I did this

    curl --pipewait -Z http://myserver/req1 http://myserver/req2
    curl: option --pipewait: is unknown

### I expected the following

It would be great if pipewait was a valid argument

### curl/libcurl version

```curl 7.67.0-DEV (x86_64-unknown-linux-gnu) libcurl/7.67.0-DEV OpenSSL/1.0.2k-fips zlib/1.2.7 brotli/1.0.1 c-ares/1.15.0 nghttp2/1.39.1
Release-Date: [unreleased]
Protocols: dict file ftp ftps gopher http https imap imaps pop3 pop3s rtsp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli Debug GSS-API HTTP2 HTTPS-proxy IPv6 Kerberos Largefile libz NTLM NTLM_WB SPNEGO SSL TrackMemory UnixSockets
```

### context

Parallel transfers made it to curl recently, and pipewait limits the connection churn. More importantly for my usage, a `--pipewait` option would make it more likely to be able to reproduce curl bugs on the command line and reduce the amount of C code necessary for bug reports. 😄

**Comment by bagder** [comment]
I agree! Can we possibly think of a better and perhaps more descriptive name?

**Comment by TvdW** [comment]
```
--prefer-multiplexing
--no-preconnect
--modern
```

Or, make it the default, and allow disabling it with `--http1-fast-connect`. Changing this default is probably still fine as the feature only landed recently, and it only impacts parallel transfers.

idk

**Comment by bagder** [comment]
Yeah, I think it makes to enable it by default for parallel transfers and provide an option to disable it...

**Comment by jay** [comment]
Fixed by #4500. Thanks.
